DotNet UWP Samples/Earthquake Simulator/Earthquake Simulator/App.xaml.cs

```diff
@@ -0,0 +1,309 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.ApplicationModel.AppService;
+using Windows.ApplicationModel.Background;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+using DataStreamer.App.Core;
+using Microsoft.DataStreamer.UWP;
+using System.ComponentModel;
+using Windows.UI.Core;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.DataStreamer.Samples.EarthquakeSimulator
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    sealed partial class App : Application
+    {
+        public static AppServiceConnection DataStreamerConnection;
+        private BackgroundTaskDeferral _appServiceDeferral;
+        public static bool IsConnectedToDataStreamerAppService;
+        private static App _instance;
+
+        private EarthquakeService _service;
+
+        /// <summary>
+        /// Initializes the singleton application object.  This is the first line of authored code
+        /// executed, and as such is the logical equivalent of main() or WinMain().
+        /// </summary>
+        public App()
+        {
+            this.InitializeComponent();
+            this.Suspending += OnSuspending;
+            _instance = this;
+
+            _service = new EarthquakeService(new EarthquakeRepository());
```

Comment:
omit blank lines

Reply:
I think bunching code up without a generous use of white space makes it hard to read.